data/appdata-xml.m4

```diff
@@ -36,7 +36,7 @@ AU_DEFUN([APPDATA_XML],
          AS_IF([test "x$enable_appdata_validate" = "xyes"],
                [AC_MSG_ERROR([AppData validation was requested but appstream-util was not found])])])
 
-  AC_SUBST([appdataxmldir], [${datadir}/appdata])
+  AC_SUBST([appdataxmldir], [${datadir}/metainfo])
```

Comment:
Oh, that valuable info! I can confirm that Arch, Debian, Ubuntu and OpenSUSE are ready for this change. I assumed Fedora was ready too, since appstream-builder supports this...